tests/integration/test_integratord/test_alerts_reading/test_alerts_reading.py

```diff
@@ -0,0 +1,348 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
```

Comment:
```suggestion
copyright: Copyright (C) 2015-2024, Wazuh Inc.
```
and in the rest of the files that you have modified

Reply:
Will do all together in another PR.

Reply:
Done!